packages/cozy-scanner/src/ModalScannerQualification.jsx

```diff
@@ -45,23 +53,10 @@ class ModalScannerQualification extends Component {
     const { onSave, t, dismissAction } = this.props
     const { qualification, filename } = this.state
     return (
-      <ExperimentalModal
-        title={t('Scan.save_doc')}
-        dismissAction={dismissAction}
-        primaryText={t('Scan.save')}
-        primaryAction={async () => {
-          pushAnalytics(qualification)
-          try {
-            await onSave(qualification, filename)
-          } catch (error) {
-            Alerter.error(t('Scan.error.generic'))
-          }
-        }}
-        primaryType={'regular'}
-        secondaryText={t('Scan.cancel')}
-        secondaryAction={() => dismissAction()}
-        secondaryType={'secondary'}
-        description={
+      <ExperimentalDialog>
```

Comment:
Actually I guess it's this Dialog that needs an `onClose`since it's not desktop only. But always adding the onClose is probably a good idea anyway.